scala3doc/src/dotty/dokka/model/api/internalExtensions.scala

```diff
@@ -111,14 +113,18 @@ extension (bound: Bound)
       }
 
 extension (m: DModule)
-  def updatePackages(op: Seq[DPackage] => Seq[DPackage]): DModule =
+  def updatePackages(op: Seq[Member] => Seq[Member]): DModule =
+    val topLevelPck = m.getPackages.get(0)
```

Comment:
Is it safe assumption?

Reply:
Yes. We create a module with single package.